.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install SDK and tool dependencies before container starts
+./dotnet.sh
```

Comment:
will this produce an error as no args are passed down and that script invokes dotnet with the passed arguments? Would it be better to invoke it with `--version`?

Reply:
Is there a better way to get the SDK installed? I looked around the `eng` folder, but I couldn't find a decent top level command. In aspnetcore, they have a `./restore.sh` command, but it doesn't appear that we have one.

Are we using `./dotnet.sh --version` anywhere else to ensure the SDK is pre-installed?

Reply:
We usually just do `./build.sh --restore` and that will install it but it will also restore the whole repository.

Reply:
You might want to check out the setup done in this app: https://github.com/bradygaster/TodoApi/tree/main/.devcontainer

Basically, include the dotnet-install shell script in the repo and then invoke that in your `postCreateCommand`.

Reply:
I've reworked this, now that we have prebuild enabled in this repo. The codespace should come with the repo already built.